src/App.vue

```diff
@@ -6,13 +6,19 @@ import { RouterView } from 'vue-router';
   <RouterView />
 </template>
 
-<style>
+<style lang="scss">
 html {
-  height: 100;
+  height: 100%;
+  width: 100%;
+  padding: 0px;
+  margin: 0px;
 }
 
 body {
-  height: 100;
+  height: 100%;
+  width: 100%;
+  padding: 0px;
+  margin: 0px;
   background-color: #101c33;
```

Comment:
You're getting closer, but now the page is doing this:

https://user-images.githubusercontent.com/80695157/176263570-1e4d1452-f9b9-47f1-9851-0fa3b9f3b124.mov

Remember that elements are nested within each other, and also that `vh` refers to the height of the visible area, it doesn't include any offsets, paddings, or margins.